accessibility/common-remediations.md

```diff
@@ -194,29 +299,66 @@ We _don’t_ do this:
 <div class=”person”><img src="/images/mrs-a-person.gif" alt="Mrs. A. Person" /><a href=”/”>Mrs. A. Person</a></div>
 ```
 
+### Brand logos
 
+#### What's the problem?
 
-### Brand logos
+A brand logo has been given incorrect alternative text. It may incorporate unnecessary information, contain a meaningless string, or fail to show parity with the image. 
+
+#### Who's affected by the problem?
+
+* Screen reader users
+* All users if the image fails to load for any reason (network problems, server issues, etc.)
+
+#### Why's it a problem?
+
+The purpose of a brand logo is for people to recognise the brand it represents. If the alt text fails to demonstrate this, then it's failed at its primary purpose. 
+
+#### How do I fix it? 
+
+The alt text for a brand logo should contain the name of the brand. If additional text is visible in the logo, that text must be included too. 
 
-For an image `alt` attribute is sufficient and more meaningful.
-No need of title attribute, use an inline image with an alt attribute for the brand logo.
+Do not:
+* Use useless information like "logo"
+* Use the alt text to add additional information or instructions
 
 We do this:
 ```html
-<a href="https://www.w3.org/"> <img src="w3c.png" alt="W3C"></a>
+<img src="w3c.png" alt="W3C">
+<img src="nature-logo.png" alt="Nature">
+<img src="bmc-logo.png" alt="BMC - Part of Springer Nature">
 ```
 
 We don't do this:
 ```html
-<a href="https://www.w3.org/"> <img src="w3c.png" title="homepage" alt="logo image"></a>
+<img src="w3c.png" alt="logo image">
+<img src="nature-logo.png" alt="Click here to go to the Nature homepage">
+<img src="bmc-logo.png" alt="BMC">
 ```
 
 ## Page titles
 
 ### Title doesn’t describe the page
 
-All page titles must clearly summarise the purpose of the page.
+#### What's the problem?
+
+The `<title>` attribute fails to describe the purpose of the page. 
```

Comment:
Would it be useful to mention somewhere in this section some mention of titles and SEO? There's been loads of people in the past writing very poor titles under the pretence that "it's best for SEO this way", which may not have even been the case, and it may impact usability and accessibility.

Reply:
oh god. can you remember any rough examples that I can use? I'll add search engines to the "who does it affect" list

Reply:
Nope, sorry :(

Reply:
Oh well 🤗 

I added some examples of the kinds of things that could go wrong and mentioned some search engine specific things. We can always add more later if we come across something else we didn't think of.

Reply:
💯